cockatrice/src/player.cpp

```diff
@@ -1962,7 +1963,16 @@ void Player::eventShuffle(const Event_Shuffle &event)
 
 void Player::eventRollDie(const Event_RollDie &event)
 {
-    emit logRollDie(this, event.sides(), event.value());
+    QStringList stringResults;
+    for (const auto &i : event.values()) {
+        stringResults.append(QString::number(i));
+    }
+    if (event.value()) {
+        // Backwards compatibility for old clients
+        stringResults.append(QString::number(event.value()));
+    }
+
+    emit logRollDie(this, static_cast<int>(event.sides()), stringResults.join(","));
```

Comment:
why make this a string? we could just have a list so we don't need to parse this thing down the line again...